src/sagemaker/jumpstart/hub/utils.py

```diff
@@ -222,3 +251,37 @@ def get_hub_model_version(
     hub_model_version = str(max(available_versions_filtered))
 
     return hub_model_version
+
+
+def _get_hub_model_version_for_marketplace_version(
+    hub_content_summaries: List[Any], marketplace_version: str
```

Comment:
Consider updating `marketplace_version` to `hub_model_version` to avoid confusion. IIUC this value is what the user passes in to `JumpStartModel`?

Reply:
Ah good point, but for this utility specifically it will use the marketplace version to determine the hub model version. As a result, I'd argue that we should keep as `marketplace_version`

Reply:
Oh right, because we've already tried to match it with the list of hub content versions. Thanks for clarifying that makes sense!